database-commons/src/main/java/io/cdap/plugin/db/connector/AbstractDBSpecificConnectorConfig.java

```diff
@@ -55,4 +63,11 @@ public int getPort() {
   public boolean canConnect() {
     return super.canConnect() && !containsMacro(ConnectionConfig.HOST) && !containsMacro(ConnectionConfig.PORT);
   }
+
+  public String getTransactionIsolationLevel() {
+    if (transactionIsolationLevel == null) {
+      transactionIsolationLevel = TransactionIsolationLevel.Level.TRANSACTION_SERIALIZABLE.name();
+    }
+    return TransactionIsolationLevel.Level.valueOf(transactionIsolationLevel).name();
+  }
```

Comment:
Can we set the `transactionIsolationLevel` to `TRANSACTION_READ_COMMITTED` for normal roles just to ensure that the role is mapped to the right serialization level, even with incorrect user input?

```
//if role is SYSDBA or SYSOP it will map to read_committed. else serialized
    return (!getRole().equals(ROLE_NORMAL)) ? TransactionIsolationLevel.Level.TRANSACTION_READ_COMMITTED.name() :
            TransactionIsolationLevel.Level.valueOf(transactionIsolationLevel).name();
```